### ISSUE TITLE
Projects implementing similar functionality

### ISSUE THREAD

**Issue by HarryR** [HarryR/shwop#5]
I'm tempted to put this on the backburner for now so I can focus on something else as a number of other projects have started implementing similar functionality, see: 

GitHub projects:

 * https://github.com/tcsiwula/Zcash-Eth-Atomic-Trades
 * https://github.com/chatch/hashed-timelock-contract-ethereum
 * https://github.com/artemii235/etomic-swap
 * https://github.com/AltCoinExchange/ethatomicswap
 * https://github.com/decred/atomicswap
 * https://github.com/Lamden/clove
 * https://github.com/republicprotocol/eth-atomic-swap - https://republicprotocol.com/
 * https://github.com/Atomicwallet
 * https://github.com/KomodoPlatform/BarterDEX
 * https://github.com/bisq-network

Websites / Services:

 * https://komodoplatform.com/decentralized-exchange/ - https://dexstats.info/
 * https://blog.lightning.engineering/announcement/2017/11/16/ln-swap.html
 * https://swap.altcoin.io/ - https://altcoin-app.herokuapp.com/educate
 * https://aphelion.org/ (NEO) - https://github.com/aphtoken/

Resources / Lists:

 * https://github.com/RobinHung/awesome-cross-blockchain
 * https://github.com/distribuyed/index
 * https://github.com/amisolution/index

Feel free to comment with additions if you find any.

Basically the problem is, do I spend time working on this repository and get it to a point where it can support HTLC, 0x, Raiden, ILP, Lightning, Bisque, AirSwap etc. and whatever other protocols seem interesting - that's a lot of work though.
